### PR TITLE
Add more info at end of config

### PR DESCRIPTION
**Pull request by jessitron**
This gives people some guidance at the end of 'atomist config' step

### CODE REVIEW

File: lib/config.ts [hunk 1]
```diff
@@ -165,6 +165,8 @@ export async function config(opts: ConfigOptions): Promise<number> {
     await writeUserConfig(userCfg);
 
     print.log(`Successfully wrote configuration ${chalk.green(cfgPath)}`);
+    print.log(`You are ready to connect a software delivery machine.
+  To create one, try 'atomist create sdm'. More info: https://docs.atomist.com/quick-start/#create-a-software-delivery-machine`);
```
Comment (ddgenome) [inline lib/config.ts]: This line should be wrapped.

Comment (cdupuis) [reply lib/config.ts]: I'm not a big fan of showing that message every time I run `config` which I do a lot. I think this should be a one time thing or be in the docs.

**Comment by jessitron** [comment]
@cdupuis Can we put this in now and make it one-time afterward?

**Comment by jessitron** [comment]
@dd what do you mean by wrapped? Do you want me to take out the indentation? ... or use newline characters? 😕

**Comment by ddgenome** [comment]
Both.

The second line starts with two spaces, why?

The second line is 122 characters long, not counting the leading spaces. Console apps should wrap text at 80.

**Pull request merged**